$NetBSD: patch-extern_Makefile,v 1.1 2018/03/14 06:27:21 dholland Exp $

Eliminate references to included ocaml-csv; use the pkgsrc one.

--- extern/Makefile~	2006-05-22 20:17:32.000000000 +0000
+++ extern/Makefile
@@ -7,7 +7,7 @@
 
 TOP = ..
 include $(TOP)/Top.Makefile
-SUBDIRSCLEANONLY = iCalendar ocaml-csv-1.0.3
+SUBDIRSCLEANONLY = iCalendar
 
 all:
 
